database/sql/src/main/scala/cromwell/database/slick/tables/WorkflowMetadataSummaryEntryComponent.scala

```diff
@@ -86,12 +87,24 @@ trait WorkflowMetadataSummaryEntryComponent {
         reduceLeftOption(_ || _)
       // Workflows can be queried by multiple labels, the reduceLeftOption ANDs together all workflows are associated
       // to all the labels from the query.
-      val labelsFilter = labelKeyLabelValues.map { case (labelKey, labelValue) =>
-          existsWorkflowIdLabelKeyAndValue(workflowMetadataSummaryEntry.workflowExecutionUuid, labelKey, labelValue) }.
+      val labelsAndFilter = labelAndKeyLabelValues.map { case (labelKey, labelValue) =>
+        existsWorkflowIdLabelKeyAndValue(workflowMetadataSummaryEntry.workflowExecutionUuid, labelKey, labelValue) }.
         reduceLeftOption(_ && _)
+      // Workflows can be queried by multiple labels, the reduceLeftOption ANDs together all workflows are associated
```

Comment:
This comment seems to need an update for the OR version

Reply:
🍝

Reply:
most of it might be pulled out a function too (DRYed copypasta)